src/cmdline/mod.rs

```diff
@@ -336,101 +287,88 @@ impl Cmdline {
     }
 }
 
+impl TryFrom<(usize, String)> for Cmdline {
+    type Error = ();
+
+    fn try_from(
+        (cmdline_capacity, cmdline_raw): (usize, String),
+    ) -> result::Result<Self, Self::Error> {
+        let (boot_args, init_args) = cmdline_raw.split_once("-- ").unwrap();
```

Comment:
Don't `unwrap` in a `TryFrom`, propagate the error instead.

Reply:
Ok, it makes sense... as long as the method's name starts with 'try'.